model-optimizer/mo/middle/passes/conv.py

```diff
@@ -22,6 +22,9 @@ def pad_op_transform(graph: Graph, match: dict):
         log.info('The pad node "{}" with pad mode "{}" cannot be fused.'.format(pad_op.soft_get('name'), pad_op.mode))
         return
 
+    if op.type == 'Pooling' and op.pool_method == 'max':
```

Comment:
use soft_get

Reply:
Attributes `type` and `pool_method` are always set, there is no need to use `soft_get`